### PR TITLE
Added randomcase method and some CSS

### PR DESCRIPTION
**Pull request by KingShimkus**
So I added the `randomcase` method which generates a 50/50 chance using the global `Math` variable, etc. So the returned string letters are randomly changed to upper or lowercase. I also updated some of the CSS, I don't know how you will feel about it. I personally think that it looks better with my additions/deletions. Also added `randomcase` to the tests. But it is like the `randomize` method in that it still can fail. I hope you enjoy my code. Love the repo. And hope to contribute more. :)

### CODE REVIEW

**Comment by KingShimkus** [comment]
Oh yeah, I also didn't want to update/edit the `README.md` until I get your approval. I know you said in `CONTRIBUTING.md` that you wanted us to update the README to but I don't know... :+1:

**Comment by salexzee** [comment]
Love your code. I have no issue with the CSS changes on the test page. And yes please update the README. Thanks for PR and the compliment on the repo.

File: src/Easify.js [hunk 1]
```diff
@@ -205,6 +205,35 @@
...
+    randomcase: function(str) {
+      // Randomize string
+      if (validateString(str)) {
+       // Split the string into an array
+        var arr = str.split('');
+        // Set some variables
+        var newString = '';
+        for (var i = 0; i < arr.length; i++) {
+          // Generate a 50/50 chance
+          if(Math.round(Math.random() * 1) == 1){
```
Comment (salexzee) [inline src/Easify.js]: Please make sure to use strict equality (===) instead of non strict equality (==).

**Comment by KingShimkus** [comment]
Okay, I added another `=` sign and added documentation to the `README`. :)

**Comment by KingShimkus** [comment]
Hey, so I updated a lot in the `tests` directory because I saw that the css was not really compatible with the right HTML id names and such, and also added highlighting for the method names in Javascript. I hope you like what I did.

**Comment by salexzee** [comment]
I'll probably be changing `test-styles.css` to `tests.css` in the next version, but for now it needs to keep the original filename to prevent merge conflicts when I merge the rest of the changes.

**Comment by KingShimkus** [comment]
Fixed things.

**Comment by KingShimkus** [comment]
Okay, I think I fixed everything. Next pull request won't be as sloppy, I swear. :)

**Pull request closed without merging**